decidim-pages/config/locales/en.yml

```diff
@@ -3,7 +3,7 @@ en:
   decidim:
     features:
       pages:
-        name: Pages
+        name: Page
```

Comment:
That's now always how features will work. The only place where that translation is useful is currently the creation dropdown. For example, the "pages" feature only implicitly creates a single page.

You're actually talking about `resources` (a feature can expose resources) and that's an abstraction we will work on afterwards.